pysmt/shortcuts.py

```diff
@@ -385,6 +383,52 @@ def BVAShr(left, right):
         of steps specified by the right BV."""
     return get_env().formula_manager.BVAShr(left, right)
 
+# String API
+def str_length(s):
```

Comment:
This follows the same naming conventions as Formula, including the documentation from there.